samples/debug/gdbstub/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _debug_sample:
+
+Debug Sample
+############
+
+Overview
+********
+
+A simple sample that can be used with qemu to show debug using gdb
+remote serial protocol capabilities.
+
+Building and Running
+********************
+
+This application can be built and executed on QEMU as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/debug
+   :host-os: unix
+   :board: qemu_x86
+   :goals: run
+   :compact:
+
+Open a new terminal and use gdb to connect to the running qemu as follows:
+
+.. code-block:: bash
+
+    gdb zephyr.elf
```

Comment:
gdb build/zephyr/zephyr.elf just to be accurate :)

Reply:
yep, can change that. I haven't done because the build directory not necessarily needs to be called `build`. But I saw other pr using that, so will follow this pattern.